File: SampleProjectForTesting/SampleProjectForTestingTests/SampleProjectForTestingTests.m
//
//  SampleProjectForTestingTests.m
//  SampleProjectForTestingTests
//
//  Created by ahimahas on 2017. 1. 24..
//  Copyright © 2017년 MS. All rights reserved.
//

#import <XCTest/XCTest.h>
#import "ObjcClass.h"
#import "SampleProjectForTesting-Swift.h"

@interface SampleProjectForTestingTests : XCTestCase

@end

@implementation SampleProjectForTestingTests

- (void)setUp {
    [super setUp];
}

- (void)tearDown {
    [super tearDown];
}

- (void)testExample {
    SwiftClass *aSwiftClass = [SwiftClass new];
    [aSwiftClass doSomething];
    
    ObjcClass *aClass = [ObjcClass new];
    [aClass doSomething];
}

@end

File: SampleProjectForTesting/SampleProjectForTestingTests/SampleProjectForTestingTests.swift
//
//  SampleProjectForTestingTests.swift
//  SampleProjectForTesting
//
//  Created by ahimahas on 2017. 1. 24..
//  Copyright © 2017년 MS. All rights reserved.
//

import XCTest
@testable import SampleProjectForTesting

class SampleProjectForTestingTests: XCTestCase {
    
    override func setUp() {
        super.setUp()
    }
    
    override func tearDown() {
        super.tearDown()
    }
    
    func testExample() {
        let aClass: SwiftClass = SwiftClass()
        aClass.doSomething()
        
        let aObjcClass: ObjcClass = ObjcClass()
        aObjcClass.doSomething()
    }
}
